### PR TITLE
added gitignore and reformatted html

### PR DESCRIPTION
**Pull request by Vishal0203**
Initial pull request. Just a reformat of the html, will add more info in later pull requests

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,244 +1,276 @@
...
-            <li class="fragment">What exactly was changed?</li>
-            <li class="fragment">We want to keep track of history</li>
-          </ul>
-        </p>
-      </section>
-      <section id="fr3">
-        <h2>Version Control comes to rescue</h2>
+</head>
+<body>
+<div class="reveal">
```
Comment (CuriousLearner) [inline index.html]: This would need some indentation, since it comes under body

Comment (Vishal0203) [inline index.html]: updated the commit

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (CuriousLearner) [inline .gitignore]: Please add new line here and I'll merge it in :)

File: index.html (hunk 1 above)
Comment (CuriousLearner) [inline index.html]: Thanks

File: .gitignore (hunk 2 above)
Comment (Vishal0203) [inline .gitignore]: done!

Comment (CuriousLearner) [inline .gitignore]: Thanks :+1:

**Pull request merged**